app/components/global/AppBar/ProfileMenu.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import config from 'config'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { Link } from 'react-router-dom'
+import { th } from '@pubsweet/ui-toolkit'
+import ExternalLink from '../../ui/atoms/ExternalLink'
+import Icon from '../../ui/atoms/Icon'
+
+const loginUrl = config.login.url
+
+const ProfileMenuButton = styled(Link)`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  padding: ${th('space.3')};
+`
+
+const ProfileIcon = styled(Icon).attrs({
+  iconName: 'User',
+  overrideName: '@pubsweet-pending.AppBar.Profile',
+})`
+  display: block;
+  fill: ${th('colorTextReverse')};
+  background-color: ${th('colorText')};
+  border-radius: ${th('borderRadius')};
+`
+
+const IconArrow = styled.div`
```

Comment:
Might it be worth moving this into `atoms`?

Since this icon isn't created in the same way as other icons and therefore doesn't live with the others, popping it into `atoms` might make it more visible for future devs. And they'll prob want to use it if/when they make the other menus in our application conform to designs